### PR TITLE
extend setup.py, add Makefile, gitignore, requirements.txt

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,10 +38,18 @@ Some heuristic methods are also employed to further improve the model prediction
...
+```
+pip install .
+# or
+make install
+```
 
 ### Models
-In order to run this tool you also need trained models. You can download our pretrained models from qurator-data.
+
+<!-- TODO add link to models -->
```
Comment (mikegerber) [inline README.md:50]: @vahidrezanezhad uploaded them to https://qurator-data.de/eynollah/ it seems.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# ocrd includes opencv, numpy, shapely, click
+ocrd >= 2.20.1
+seaborn >= 0.11.0
```
Comment (mikegerber) [inline requirements.txt:3]: seaborn is a plotting library, may be worth checking why this is needed

File: sbb_newspapers_org_image/__init__.py [hunk 3]
```diff
@@ -1 +1 @@
-from eynollah import *
+
```
Comment (mikegerber) [inline sbb_newspapers_org_image/__init__.py:1]: Does this belong in this PR?

**Review by mikegerber — COMMENTED** [review]
LGTM, mostly

File: requirements.txt (hunk 2 above)
Comment (kba) [reply requirements.txt:3]: For plotting 😆 Mostly for debugging, can be controlled with CLI flags.

File: sbb_newspapers_org_image/__init__.py (hunk 3 above)
Comment (kba) [reply sbb_newspapers_org_image/__init__.py:1]: Either this PR or the refactoring PR. Top-level wildcard import is suspicious here and probably unintentional.

File: README.md (hunk 1 above)
Comment (kba) [reply README.md]: This has been corrected in main, I merged now, link to qurator-data is fixed.

File: requirements.txt (hunk 2 above)
Comment (mikegerber) [reply requirements.txt:3]: :+1:

File: sbb_newspapers_org_image/__init__.py (hunk 3 above)
Comment (mikegerber) [reply sbb_newspapers_org_image/__init__.py:1]: OK

**Pull request merged**